src/styled/developer-topics-menu.ts

```diff
@@ -0,0 +1,17 @@
+import styled from '@emotion/styled';
+import { colorMap } from './theme';
+
+const DeveloperTopicsMenuWrapper = styled('div')`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 50%;
```

Comment:
Any reason we are using `percentage` values here versus leveraging `flex` to determine the amount of space to take?